lib/improver/tests/utilities/cube_manipulation/test__associate_any_coordinate_with_master_coordinate.py

```diff
@@ -46,8 +46,25 @@
 from improver.tests.ensemble_calibration.ensemble_calibration.\
     helper_functions import set_up_temperature_cube
 
-from improver.tests.utilities.cube_manipulation.\
-    helper_functions import check_coord_type
+
+def check_coord_type(cube, coord):
```

Comment:
Is there are a reason this was moved here? Might this be used elsewhere in which case it might be best not to be defined within a unit test?

Reply:
This is the only place this function was being used, and moving it to this file allowed the otherwise completely redundant `cube_manipulation.helper_functions` module to be deleted.  If there is a use case we could consider collating useful little functions like this (which are defined far and wide, and probably duplicated, throughout the unit tests) into a general `tests.helper_functions` module.  But I think that would be out of scope for this ticket.